src/frontend/app/shared/components/add-service-instance/specify-details-step/specify-details-step.component.ts

```diff
@@ -400,8 +409,16 @@ export class SpecifyDetailsStepComponent implements OnDestroy, AfterContentInit
       switchMap(o => create$),
       filter(a => !a.creating),
       switchMap(a => {
-        if (a.error) {
-          return create$;
+
+        if (isEditMode) {
+        const updating = a.updating[UpdateServiceInstance.updateServiceInstance];
+          if ( !!updating && updating.error) {
+            return create$;
+          }
+        } else {
+          if (a.error) {
+            return create$;
+          }
```

Comment:
We're returning `create$` in these two cases. Can this be simplified into one if statement?